firstPart/3_convertNumber.js

```diff
@@ -0,0 +1,32 @@
+const numbers = {
+    "I": 1,
+    "V": 5,
+    "X": 10,
+    "L": 50,
+    "C": 100,
+    "D": 500,
+    "M": 1000
+}
+
+const convertRomanToArabicNum = (romanNum) => {
+    const arr = romanNum.split('');
+    let arabicNum = 0;
+    for (let i = 0; i < arr.length; i++) {
+        const first = numbers[arr[i]];
+        const second = numbers[arr[i + 1]] || 0;
+        first >= second
+            ? arabicNum += first
+            : (arabicNum += second - first, i++);
```

Comment:
using ternary in cases like that is not a good idea, just because code become less readable and also. Use if statements instead.
Ternary are good when you have cases like that:
`const someVariable = statement ? <value1> : <value2>`, because ternary main meaning is that we want to return one of option

Reply:
Ok, initially I made it with "if else", but after that converted it that way , because I was thinking that you would like ternary more. for this case. Ok, my bad, I will use 'if statement".